### DISCUSSION TITLE
How to Verify Existing Plot Files and How to Resume Interrupted Plotting / Unfinished Plots

### DISCUSSION THREAD

**Discussion by InfinitePotential2020** [Chia-Network/chia-blockchain#535] [Q&A] [answered]
Hello!

I would like to know How to Verify Existing Plot Files in order to confirm if they are 100% Valid and uncorrupted and How to Resume an Interrupted Plotting / Unfinished Plots when the Chia GUI Software was accidentally closed.

Thank You!

**Comment by hoffmang9** [comment, marked as answer]
There is no resume of plotting yet. The GUI verifies all plots with each set of challenges. To specifically verify all or individual plots you can use `chia plots verify`

**Reply by BestNgineering** [reply]
Sorry to ask but, how long do you think it is going to take to develop "resume" feature

**Comment by DarthJahus** [comment]
Pause and resume in Plots would be an amazing addition.

**Reply by markusuk1** [reply]
So my Gui Crashed during plotting at 100% during the process of copying the files across to the final directory.  Are these files complete?  So can I just delete the the .tmp part of the file-name and they will work then coppy them from my plotter to my Final drive manualy otherwise ive just wasted 8 hours of plotting.

**Reply by utopianit-base** [reply]
I've done this before and had success. It ended with *plot.2.tmp or something and I changed it to *.plot after it failed to copy (why not move?!). Moved the file over via a different method and the Farmer saw the new file and was OK.

**Comment by mist0706** [comment]
Pausing and resuming paused plots is possible in linux using the built in task manager. It wont be possible to pause, restart and then resume though. The state of the process is just set to sleep.

**Comment by DarthJahus** [comment]
Hibernation (not hybrid sleep) works well on Windows. Of course, if you have "scheduled" queues, just remove all pending jobs and leave only currently _plotting_ plots. After you wake up your computer, it will continue normally.

**Comment by room101-dev** [comment]
WHat I see is when the NMVE overheats the 'chia create plot' just terminates itself, often without even writing to the log.

Watch the temp, and watch 'sudo iotop' if its goes to zero, you got problems, watch the 'htop' if the processes fall off the top, its means the NVME controller has quit responding.

Once a plot fails, usually 5th hour in a 6hr process, nothing can bring them back, its just the way this garbage software is written, only purpose is to KILL HW.

**Comment by jayhohoho2019** [comment]
with chia plots check, what is the reasonable Proofs ratio range?  How do I conclude a plot file is "normal?"  Between 0.8 - 1.2?
